How_to_Release.md

```diff
@@ -0,0 +1,23 @@
+# Creating a new release
+
+This is a guide how to create a new ruff release
+
+1. Update the version with `rg 0.0.269 --files-with-matches | xargs sed -i 's/0.0.269/0.0.270/g'`
+1. Update [BREAKING_CHANGES.md](BREAKING_CHANGES.md)
+1. Create a PR with the version and BREAKING_CHANGES.md updated
+1. Merge the PR
+1. Run the release workflow with the tag (without starting `v`) as input. Make sure main has your
```

Comment:
```suggestion
1. Run the release workflow with the version number (without starting `v`) as input. Make sure main has your
```

Nit: Rephrase to version number? I first thought I would need to create the tag myself and then provide its name.